packages/css-framework/src/components/_radio-card.scss

```diff
@@ -0,0 +1,42 @@
+@use "../abstracts/functions" as f;
+
+$animation-timing: 0.2s;
+$radio-focus-width: 0.1rem;
+
+.rn-radio-card {
+  display: inline-flex;
+  flex-direction: column;
+  width: 22rem;
+  padding: f.spacing("10");
+  margin: f.spacing("4");
+  border-radius: 4px;
+  border: 1px solid f.color("neutral", "100");
+  outline: none;
+
+  &:hover {
+    border-color: f.color("action", "400");
+    box-shadow: 0 0 0 3px f.color("action", "100");
+    cursor: pointer;
+
+    .rn-radio input ~ .rn-radio__checkmark {
```

Comment:
I attempted to programmatically simulate a hover on the child radio via a ref but it wouldn't work for some reason. As such i've ended up duplicating a couple of hover styles here instead.

Along the lines of this:

```typescript
  const handleMouseEnter = (_: React.MouseEvent<HTMLDivElement>) => {
    ref.current.dispatchEvent(new MouseEvent('moustenter'))
  }
```